frontend/apps/filemanager/filemanagerutil.lua

```diff
@@ -29,12 +29,36 @@ function filemanagerutil.abbreviate(path)
     return path
 end
 
--- Purge doc settings in sidecar directory,
+-- Purge doc settings in sidecar directory
 function filemanagerutil.purgeSettings(file)
     local file_abs_path = util.realpath(file)
     if file_abs_path then
         DocSettings:open(file_abs_path):purge()
     end
 end
 
+-- Purge doc settings except kept
+function filemanagerutil.purgeViewSettings(file)
+    local settings_kept = {
+        bookmarks = true,
+        bookmarks_sorted = true,
+        bookmarks_version = true,
+        cre_dom_version = true,
+        highlight = true,
+        highlights_imported = true,
+        last_page = true,
+        last_xpointer = true,
```

Comment:
Have you had another look at some of your setting files ? Or did you just trust my quick look ? :)

Reply:
It's my list.